qa/tests/build.xml

```diff
@@ -231,14 +239,14 @@
                debug="on" deprecation="on" includeantruntime="true">
 
             <src path="${org.jboss.jbossts.qa.src}"/>
-            <src path="${jts.build.dir}/jts/tests/classes"/>
-            <src path="../../common/tests"/>
+            <src path="${external.libraries}/lib/jts-test-sources/com/"/>
+            <src path="${external.libraries}/lib/common-test-sources/com/"/>
 
             <exclude name="**/javaidl/**/*.java" unless="is.sunorb"/>
 
             <exclude name="org/jboss/jbossts/qa/ArjunaCore/Common/ClassPathParserTest.java" />
-            <exclude name="com/hp/mwtests/ts/jts/recovery/RecoveredTransactionUnitTest.java" />
-            <exclude name="com/hp/mwtests/ts/jts/recovery/RecoveredServerTransactionUnitTest.java" />
+            <exclude name="hp/mwtests/ts/jts/recovery/RecoveredTransactionUnitTest.java" />
```

Comment:
How come the com/ could be removed from this line and the one below?

Reply:
IIRC, packing and unpacking source code ended up with a bit of headache for `ant`. To fix this, I had to remove `com` from the path. I can double check this though and see if I manage to introduce `com` back in the path

Reply:
I don't think this is a problem but I'll double check if I can restore `com` in the path. Thanks Tom

Reply:
Thank you